components/brave_wallet/common/hash_utils.cc

```diff
@@ -35,8 +36,8 @@ std::string Namehash(const std::string& name) {
   std::vector<std::string> labels =
       SplitString(name, ".", base::KEEP_WHITESPACE, base::SPLIT_WANT_NONEMPTY);
 
-  for (auto rit = labels.rbegin(); rit != labels.rend(); rit++) {
-    std::string label_hash = KeccakHash(*rit, false);
+  for (const auto& rit : base::Reversed(labels)) {
```

Comment:
`rit` -> `label`

Reply:
Fixed in 6720219d61654b18ecae768debeee9b0ad1cf2cb.